html-forms/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>HTML forms</title>
+</head>
+
+<body>
+  <h1>Coding Form!</h1>
+
+  <form id="Coding Form">
+    <label for="Email">Email</label><br>
+    <input id="Email" type="text" name="form" required><br>
+    <label for="dropown">Why is coding cool?</label><br>
```

Comment:
I would avoid using `br` tags. If you want to have the input or label break onto the next line, you want to wrap it in a `div`. `br` tags are only really used for text formatting for things like poetry.